lib/beaker/dsl/install_utils.rb

```diff
@@ -1443,9 +1446,12 @@ def copy_module_to(one_or_more_hosts, opts = {})
               #rename to the selected module name, if not correct
               cur_path = File.join(target_module_dir, source_name)
               new_path = File.join(target_module_dir, module_name)
-              if cur_path != new_path
-                # NOTE: this will need to be updated to handle powershell only windows SUTs
-                on host, "mv #{cur_path} #{new_path}"
+              if (cur_path != new_path)
+                if (host["platform"] !~ /windows/ || host.is_cygwin?)
```

Comment:
You can just do if host.is_cygwin? - non-windows and all other platforms are considered to be cygwin like and will pass through.